dissect/target/loaders/tar.py

```diff
@@ -196,11 +197,28 @@ def is_tar_magic(path: Path, magics: Iterable[bytes]) -> bool:
         return False
 
     with path.open("rb") as fh:
-        headers = [fh.read(6)]
-        fh.seek(257)
-        headers.append(fh.read(8))
+        # The minimum file size of an uncompressed tar is 512 bytes,
+        # but a compressed tar file could be smaller.
+        try:
+            fh.seek(0)
+            buf = fh.read(tf.BLOCKSIZE)
+        except EOFError:
```

Comment:
`fh.read` can't cause an `EOFError` (unless when running into a bug like this https://github.com/fox-it/dissect.target/issues/1115), right? The buffer read will just be smaller than `tf.BLOCKSIZE`.